src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promis1 = new Promise(function(resolve) {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+promis1.then((res) => {
+  message(res, 'message');
+});
+
+const promis2 = new Promise(function(resolve, reject) {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promis2.catch((res) => {
+  message(res, 'error-message');
+});
+
+function message(result, type) {
```

Comment:
```suggestion
function appendMessage(result, type) {
```
function name should describe an action - what it does
[info](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx)